model/generate.go

```diff
@@ -158,13 +158,24 @@ func (g *GeneratedProject) NewVersion() (*Project, *ParserProject, *Version, *ta
 }
 
 func (g *GeneratedProject) Save(ctx context.Context, p *Project, pp *ParserProject, v *Version, t *task.Task, pm *projectMaps) error {
+
+	//if pp.ConfigUpdateNumber > v.ConfigUpdateNumber { // I don't think this could happen until UseParserProject, and even then maybe not until we stop preferencing version
+	//	update parser project contingent on equal config numbers
+	//	update version
+	//
+	//} else { // this is the current case
+	//	update version contingent on equal config numbers
+	//	update parser project
```

Comment:
I assume this update parser project would set the version number?

Reply:
Update version would set config and config update number, and update parser project would update all of the parser project fields including config update number (i.e. the existing updateParserProjectWithConfig was intended to do this).